butterfly/QueryLayer/TileQuery.py

```diff
@@ -76,3 +76,11 @@ def target_origin(self):
     def source_bounds(self):
         return self.all_scales * self.target_bounds
 
+    @property
+    def preload_source(self):
+        loader = self.source_class
+        block, dtype = loader.preload_source(self)
+        self.RUNTIME.IMAGE.BLOCK.VALUE = block
+        return [block, dtype]
+
```

Comment:
Basically just calls the `preload_source` method of the relevant `DataSource` class.